bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/InpatientClaimGroup.java

```diff
@@ -213,9 +213,9 @@
 	public List<IcdCode> diagnosesExternal = new LinkedList<>();
 
 	/**
-	 * Represents the data contained in {@link Column#LINE_NUM} and subsequent
-	 * columns: one entry for every "claim line" in the claim represented by
-	 * this {@link InpatientClaimGroup} instance.
+	 * Represents the data contained in {@link Column#CLM_LINE_NUM} and
+	 * subsequent columns: one entry for every "claim line" in the claim
+	 * represented by this {@link InpatientClaimGroup} instance.

```

Comment:
I repeated it because I was having the same build error.  Confused why it didn't pick up my merged change from earlier today?